src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -1191,7 +1457,7 @@ await using (var context = InMemoryApplicationDbContext(contextId))
                             ReleaseName = "2035",
                             TimePeriodCoverage = TimeIdentifier.March,
                             PreReleaseAccessList = "New access list",
-                            ApprovalStatus = ReleaseApprovalStatus.Draft,
+                            ApprovalStatus = ReleaseApprovalStatus.HigherLevelReview,
```

Comment:
Why did these need to change out of interest?

Reply:
To prevent the bug when trying to create an prerelease access list from occurring, the UpdateRelease method still needs to only add a ReleaseStatus if the ApprovalStatus has changed - even though nothing using that endpoint will change the ApprovalStatus.

Once UpdateRelease is updated as part of the fixup ticket after this PR is merged, ReleaseUpdateViewModel will no longer take an ApprovalStatus and so this will no longer be an issue. So, to finally answer your question, I needed to do this to make the test pass. :)